htsget.md

```diff
@@ -357,15 +357,119 @@ Initial guidelines, which we expect to revise in light of future experience:
 
 The data block URL and headers might contain embedded authentication tokens; therefore, production clients and servers should not unnecessarily print them to console, write them to logs, embed them in error messages, etc.
 
+# POST requests
+
+In addition to the GET method, servers may optionally also accept POST requests.
+The main differences to the GET method are that query parameters are encoded in JSON format and it is possible to request data for more than one genomic range.
+The data returned is a JSON "ticket", as described for the GET method.
+
+Htsget POST requests must be "safe" as defined in section 4.2.1 of [RFC 7231], that is they must be essentially read-only.
+
+## URL parameters
+
+POST and GET requests should use the same URLs.
+
+## Query parameters
+
+The query parameters are similar to those used in the GET request, formatted into a JSON object.
+The `referenceName`, `start` and `end` parameters are not used for POST requests as they are replaced by `regions` which allows more that one range to be specified.
+
+<table>
+<tr markdown="block"><td>`format`  _optional string_</td><td>
+Request read data in this format. Default: BAM. Allowed values: BAM,CRAM.
+
+The server SHOULD reply with an `UnsupportedFormat` error if the requested format is not supported.
+[^a]
+</td></tr>
+<tr markdown="block"><td>`fields`  _optional array of strings_</td><td>
+List of fields to include.  See Field filtering.
+  <table>
+     <tr markdown="block"><td>Field name _string_</td></tr>
+  </table>
+</td></tr>
+<tr markdown="block"><td>`tags`  _optional array of strings_</td><td>
+List of tags to include, default: all.  If an empty array is specified, no tags are included.
+
+The server SHOULD respond with an `InvalidInput` error if `tags` and `notags` intersect.
+  <table>
+     <tr markdown="block"><td>Tag name _string_</td></tr>
+  </table>
+</td></tr>
+<tr markdown="block"><td>`notags`  _optional array of strings_</td><td>
+List of tags to exclude, default: none.  If an empty array is specified, no tags are included.
+
+The server SHOULD respond with an `InvalidInput` error if `tags` and `notags` intersect.
+  <table>
+     <tr markdown="block"><td>Tag name _string_</td></tr>
+  </table>
+</td></tr>
+<tr markdown="block"><td>`regions` _object_</td><td>
+Regions to return.  If not present, the entire file will be returned.  If an empty object or null, just the header will be returned.
```

Comment:
This is a good idea for allowing people to get the header.

Reply:
That is nice, but I think we should still come up with a way to get the header with a GET request.